### PR TITLE
Onboarding UI changes

### PR DESCRIPTION
**Pull request by SzymonGorowski**
https://trello.com/c/nSyUxjxm/727-onboarding-ui-changes

### CODE REVIEW

File: AirCasting/Onboarding/NearAirDescription.swift [hunk 1]
```diff
@@ -6,15 +6,15 @@ import AirCastingStyling
 
 struct NearAirDescription: View {
     var completion: () -> Void
+    
     var body: some View {
-        VStack(alignment: .leading, spacing: 25) {
+        VStack(alignment: .leading) {
```
Comment (anna1901) [inline AirCasting/Onboarding/NearAirDescription.swift:11]: Could you horizontally center the button on this screen as well? It's a bit to the left

File: AirCasting/Onboarding/PrivacyOnboarding.swift [hunk 2]
```diff
@@ -73,7 +72,7 @@ private extension PrivacyOnboarding {
             .font(Fonts.boldTitle1)
             .foregroundColor(.accentColor)
             .multilineTextAlignment(.leading)
-            .padding(.bottom, 20)
+            .padding(.bottom, 55)
```
Comment (anna1901) [inline AirCasting/Onboarding/PrivacyOnboarding.swift:75]: Same here, I think the title looks better with a 24 padding

File: AirCasting/Onboarding/NearAirDescription.swift [hunk 3]
```diff
@@ -26,7 +26,7 @@ private extension NearAirDescription {
     private var progressView: some View {
         ProgressView(value: 0.2)
             .accentColor(.accentColor)
-            .padding(.bottom, 20)
+            .padding(.bottom, 30)
```
Comment (anna1901) [inline AirCasting/Onboarding/NearAirDescription.swift:29]: Are you sure we need this padding under the progress view?

File: AirCasting/Onboarding/AirBeamOnboarding.swift [hunk 4]
```diff
@@ -70,7 +71,7 @@ private extension AirBeamOnboarding {
     private var progressBar: some View {
         ProgressView(value: 0.4)
             .accentColor(.aircastingMint)
-            .padding(.bottom, 20)
+            .padding(.bottom, 30)
```
Comment (anna1901) [inline AirCasting/Onboarding/AirBeamOnboarding.swift:74]: Are you sure we need this padding under the progressView?

**Review by anna1901 — COMMENTED** [review]
Great changes. 😊 I think the button on the GetStarted view should also be aligned with the rest.

File: AirCasting/Onboarding/AirBeamOnboarding.swift (hunk 4 above)
Comment (SzymonGorowski) [reply AirCasting/Onboarding/AirBeamOnboarding.swift:74]: I think the padding is necessary because on the design we have 30/40px of padding.

**Review by jankrzempek — COMMENTED** [review]
The only thing is `Get Started` button. Could you please align its height so it matches other buttons?

**Review by SzymonGorowski — COMMENTED** [review]
The position of the GetStarted button has been adjusted.

**Review by jankrzempek — APPROVED** [review]
This UI is now outstanding, you did it Szymon, so well 🥳

File: AirCasting/Onboarding/NearAirDescription.swift (hunk 1 above)
Comment (anna1901) [reply AirCasting/Onboarding/NearAirDescription.swift:11]: This button is actually a navigation link, and for some reason, it doesn't take up the whole space, so there is a bigger gap between the button and the edge of the screen on the right thank on the left. Maybe you'll be able to see it when you change the alignment to `.center` and see it move a bit.
Screenshot 2022-07-05 at 10 39 50
Screenshot 2022-07-05 at 10 40 14

Comment (anna1901) [reply AirCasting/Onboarding/NearAirDescription.swift:11]: But probably the alignment of the VStack shouldn't be changed and just the button should be centered (or bigger)

**Pull request merged**